css-view-transitions-2/Overview.bs

```diff
@@ -593,54 +660,23 @@ When capturing the old or new state for an element, perform the following steps
-
-			1. If |outboundTransition|'s [=ViewTransition/phase=] is "`done`", then call |onReady| and return.
-
-			1. Assert: |outboundTransition|'s  [=ViewTransition/phase=] is "`pending-capture`".
-
-			1. [=Clear view transition=] |outboundTransition|.
-
-				Note: The ViewTransition object on the old Document should be destroyed after its state has been copied to the new Document below.
-					We explicitly clear it here since the old Document may be cached by the UA.
-
-			1. Set |oldDocument|'s [=auto-skip view transitions=] to false.
-
-			1. [=Queue a global task=] on the [=DOM manipulation task source=] given |newDocument|'s [=relevant global object=],
-				to perform the following steps:
-
-				1. Let |inboundTransition| be a new {{ViewTransition}} in |newDocument|'s [=relevant Realm=],
-					whose [=ViewTransition/named elements=] is |outboundTransition|'s [=ViewTransition/named elements=],
-					[=ViewTransition/initial snapshot containing block size=] is |outboundTransition|'s [=ViewTransition/initial snapshot containing block size=],
-					and whose [=ViewTransition/is inbound cross-document transition=] is true.
+			whose [=ViewTransition/active types=] is |resolvedRule|.
 
-				1. Set |newDocument|'s [=active view transition=] to |inboundTransition|.
+		1. Set |outboundTransition|'s [=outbound post-capture steps=] to the following steps given a [=view transition params=]-or-null |params|:
+			1. Set |newDocument|'s [=inbound view transition params=] to |params|.
 
-				1. [=Resolve=] |inboundTransition|’s [=ViewTransition/update callback done promise=] with undefined.
+				Note: The inbound transition is activated after the dispatch of {{Window/pagereveal}} to ensure mutations made in this event apply to the captured new state.
 
-				1. Set |inboundTransition|’s [=ViewTransition/phase=] to "`update-callback-called`".
+			1. Call |proceedWithNavigation|.
 
-				1. Call |onReady|.
-					
-					Note: The |inboundTransition| is activated after the dispatch of {{Window/pagereveal}} to ensure mutations made in this event apply to the captured new state.
-
-				1. At any given time, the UA may decide to skip |inboundTransition|, e.g. after an [=implementation-defined=] timeout.
-					To do so, the UA should [=queue a global task=] on the [=DOM manipulation task source=] given |newDocument|'s [=relevant global object=] to perform the following step:
-						If |transition|'s [=ViewTransition/phase=] is not "`done`", then [=skip the view transition=] |transition| with a "{{TimeoutError}}" {{DOMException}}.
+		1. Set |oldDocument|'s [=active view transition=] to |outboundTransition|.
 
-			Note: |outboundTransition| is not exposed to JavaScript, it is used only for capturing
-			the state of the old document.
+			Note: The process continues in [=perform pending transition operations=].
 
-		1. Set |oldDocument|'s [=active view transition=] to |outboundTransition|.
+		1. React to |outboundTransition|'s [=ViewTransition/finished promise=]:
```

Comment:
I really don't like this. We're just making our spec unnecessarily complicated by relying on promises for browser steps. I also suggested adding an "addition to skip steps" style of update for this because that's what we'll do when the specs are eventually merged.